spec/views/spotlight/roles/index.html.erb_spec.rb

```diff
@@ -22,8 +22,7 @@
       assert_select "tr[data-edit-for='#{admin_role.id}']"
       assert_select 'td', /jane@example.com/
       assert_select 'td', 'Admin'
-      assert_select "input[type='submit'][data-behavior='destroy-user'][data-target='#{admin_role.id}']," \
-                    "input[type='submit'][data-behavior='destroy-user'][data-bs-target='#{admin_role.id}']"
+      assert_select "input[type='submit'][data-behavior='destroy-user'][data-target='#{admin_role.id}']"
```

Comment:
these are not bootstrap targets so we should just use `data-target` not `data-bs-target`